docs/components/connectors/use-connectors/index.md

```diff
@@ -63,6 +63,26 @@ These fields export responses from external **Connector** calls into process var
 This field declares a singular process variable designated for the export of responses from a **Connector** call.
 The resulting process variable can be subsequently utilized within the ongoing process.
 
+#### Example
+
+If you set `result` inside the `Result variable` field of the REST outbound Connector, this variable is available:
+
+```json
+{
+  "result": {
+    "status": 200,
+    "headers": {
+      "date": "Thu, 03 Apr 2025 07:05:19 GMT",
+      "server": "nginx",
+      "content-type": "text/html; charset=UTF-8"
+    },
+    "body": "<h1>Hello, World!</h1>",
```

Comment:
AFAIK customer usually call REST APIs that return JSON. I would prefer to have an example for a simple JSON response: 
```
{ 
  "orderNumber": "1234",
  "date": "2025-04-01",
  "customerId": "567",
  "address": { ... }
}
```